FirebaseSessions/Tests/Unit/Mocks/MockInstallationsProtocol.swift

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import FirebaseInstallations
+@_implementationOnly import FirebaseInstallations
```

Comment:
Why is installations implementationOnly? Is this because it is purely internal and not exposed as public?

Reply:
Yeah exactly - so it's not exposed to our users. Not a huge deal because we're just a dependency of Crash / Perf but is good practice.